int8 SQUARE_WAVE=0
int8 SINE_WAVE=1
int8 STOP=2

int8 type